pyproject.toml

```diff
@@ -73,6 +76,7 @@ ipython = "^7.22.0"
 
 [tool.poetry.scripts]
 det-sal-on-img-dir = "xaitk_saliency.utils.bin.det_sal_on_img_dir:det_sal_on_img_dir"
```

Comment:
With the removal of the defunct tool we need to remove this tool script export as well.